### PR TITLE
11.0 doc data files update pat

### PR DESCRIPTION
**Pull request by pat-odoo**
Data Files doc Improvements.
https://www.odoo.com/documentation/11.0/reference/data.html

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: doc/reference/data.rst [hunk 1]
```diff
@@ -23,13 +23,28 @@ of an XML data file is the following:
...
 
 Data files are executed sequentially, operations can only refer to the result
 of operations defined previously
 
+Odoo Tag Parameter
+------------------
+
+``noupdate``
+   ``values : 0 or 1``
+    As name suggest, ``noupdate="0"`` mode make record can be modify by other advanced modules otherwise not with ``noupdate="1"``.
```
Comment (Elkasitu) [inline doc/reference/data.rst]: As the name suggests, ``noupdate="0"`` mode makes the record modifiable by other modules. ``noupdate="1"``, on the other hand, makes the record unmodifiable.

Comment (pat-odoo) [reply doc/reference/data.rst]: PR Updated. Thanks!

**Review by melroy89 — APPROVED** [review]

**Comment by rdeodoo** [comment]
@rco-odoo Since you are the most active on that file :yum: 
Not sure who could check this one, and there is no point to leave it open forever

**Comment by yelizariev** [comment]
docs about `noupdate` is already added to v14 https://github.com/odoo/odoo/commit/ffc69127f8f91e79a1263a0ab5685e67d93d569a

Examples may be obsolete and I'd prefer just to search examples in odoo source, but if you like, you could make a new PR to v14 branch.

This PR could be closed

**Comment by C3POdoo** [comment]
Dear @pat-odoo,

Thank you for your contribution but the version 11.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**